### ISSUE TITLE
Create an Open edX contributor's guide

### ISSUE THREAD

**Issue by BbrSofiane** [openedx/wg-build-test-release#11]
Andrés González  is writing [Open edX contributor's guide](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1682211674/Open+edX+contributor+s+guide).

https://openedx.atlassian.net/browse/BTR-9

**Comment by arbrandes** [comment]
@BbrSofiane, can we add @angonz as an assignee, here?  I think we'll need to add him to the org or project, somehow.

**Comment by BbrSofiane** [comment]
@arbrandes  We can add him to the org but I think once he comments on the issue we can add him as assignee. I had the same issue with Felipe but after he replied I was able to assign the issue to him.

**Comment by arbrandes** [comment]
Ah, ok, great.  @angonz, you there? ;)

**Comment by angonz** [comment]
Hi there! I'm here!

**Comment by ashadunga** [comment]
Hi Team,

Can I be part of this documentation? Technically had 4+ years of experience in working on Projects developed using Python, Java, SQL and ETL . With over 2+ years of experience in Documenting the technical details of the project.

**Comment by BbrSofiane** [comment]
@angonz is this still something that's on your radar?

In the last BTR meetup, the idea was brought up to makes this a community effort. The assignee of this issue will ask each BTR team member to contribute and make sure they do so in a timely manner.

**Comment by angonz** [comment]
Hi @BbrSofiane! Thanks for calling me. Yes, I'm still here. Unfortunately I have another appointment at the same time as the contributors meetings, so I'm unable to attend currently.  But I'm still on the road... Now struggling with Tutor and Kubernetes :-S

**Comment by clemente** [comment]
Hi, the guide at [BTR-9](https://openedx.atlassian.net/browse/BTR-9) seems already good. @angonz, what's missing to finish it and finish the task?
- do you need a review from people who are new contributors?
- and/or a review from expert contributors?

There's @BbrSofiane's [comment](https://github.com/openedx/build-test-release-wg/issues/11#issuecomment-908468200) about requesting others to expand that guide. And [with already an offer](https://github.com/openedx/build-test-release-wg/issues/11#issuecomment-870259731).

(I'm looking at the remaining work in BTR tasks, since I'm looking for contributions tasks for new members in OpenCraft).

**Comment by BbrSofiane** [comment]
@clemente Thank you for volunteering! I think this is something @sarina is looking at so we can probably close this issue.

In terms of issues where you can contribute, have a look at #67 if you have experience with react. Otherwise, feel free to join us on slack and at one of our bi-weekly meetups to get up to date on the latest issues we're looking to tackle!

**Comment by sarina** [comment]
@BbrSofiane this isn't something I'm actively working on. There's also no acceptance criteria here (what is the intention of the ticket? Audience? What should they know at the end of reading documentation? etc), so I can't tell (to @clemente 's point) if the guide at https://openedx.atlassian.net/wiki/spaces/COMM/pages/1682211674/Open+edX+Platform+contributor+s+guide is complete and just in need of review, or if it needs something else.

Could you summarize where we're at with this ticket?

**Comment by clemente** [comment]
@BbrSofiane By the way, I'm not contributing to do it myself, but I'm looking for tasks to assign to new developers. I looked at #67 and it seems complex for someone new to Open edX since it depends on architecture changes. I'll watch it. (And I'll coordinate about the meetings; I think some from OpenCraft are already attending them).

About this task: I'll let you comment. It looks like https://openedx.atlassian.net/wiki/spaces/COMM/pages/1682211674/Open+edX+Platform+contributor+s+guide needs to be reviewed (there's a TODO) and then it could be announced somewhere (forum?) and linked from more pages.

**Comment by NeOneSoft** [comment]
Hi @clemente!!!, following this issue, I was wondering if you have any update or you were available to assign some tasks to new developers on your team. Thanks in advance!!

**Comment by clemente** [comment]
@NeOneSoft Hi, unfortunately there aren't any updates from me or OpenCraft regarding this task. I recently left OpenCraft, and the role to find easy tasks for junior developers [seems to not have been assigned yet](https://forum.opencraft.com/t/junior-developer-tasks-new-assignee-needed/1271).

This means that if anyone is interested, they can continue this task. As mentioned in previous comments, the remaining work seems to be reviewing https://openedx.atlassian.net/wiki/spaces/COMM/pages/1682211674/Open+edX+Platform+contributor+s+guide and announcing it or making it „more“ official.

**Comment by NeOneSoft** [comment]
Ok @clemente thank you so much for your response. @BbrSofiane do we consider to ask for help on opencraft to re-assign this issue from their said or do you think that we can ask for help from others in the community? Thank's again

**Comment by arbrandes** [comment]
This is being worked on by Sarina.  We can close this one.

**Comment by regisb** [comment]
I propose the following acceptance criteria for this ticket:

Modify [CONTRIBUTING.rst](https://github.com/openedx/edx-platform/blob/master/CONTRIBUTING.rst) such that:
- it links to the [Open edX platform contributor guide](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1682211674/Open+edX+Platform+contributor+s+guide)
- it points new contributors to the working groups: both the [page on Atlassian](https://openedx.atlassian.net/wiki/spaces/COMM/pages/46793351/Open+edX+Working+Groups) and the [forum category](https://discuss.openedx.org/c/working-groups/).

**Comment by mariajgrimaldi** [comment]
@jalondonot: should we update the issue description with the criteria Regis is proposing? I believe we should do it since the previous developer working on this said those tasks were what was left to this.

**Comment by jalondonot** [comment]
@mariajgrimaldi, don't the courses we're creating cover this issue?

**Comment by sarina** [comment]
I think it would be good to have a summarized contributor guide that points at the course. People who have already taken the course or who are somewhat familiar with the project will have an easier time skimming / searching a concise guide that lays out the main points.

Let's fold this into our courses work. We can aim to update the already-existing wiki page to match the structure of the contributing course. Does that sound reasonable @jalondonot @mariajgrimaldi ?

**Comment by mariajgrimaldi** [comment]
@sarina: should we close this issue now that the dev course is out?

**Comment by sarina** [comment]
I think so, yes!
